packages/web-components/fast-components/docs/api-report.md

```diff
@@ -57,7 +57,7 @@ import { NumberFieldOptions } from '@microsoft/fast-foundation';
 import { Picker } from '@microsoft/fast-foundation';
 import { PickerList } from '@microsoft/fast-foundation';
 import { PickerListItem } from '@microsoft/fast-foundation';
-import { PickerMenu } from '@microsoft/fast-foundation';
+import { PickerMenu as PickerMenu_2 } from '@microsoft/fast-foundation';
```

Comment:
"PickerMenu_2" ?

Reply:
Setting the fill color of a flyout requires creating a subclass of the foundation component. This is how the API report represents both the foundation and the specific components.